packages/components-js/tests/vrt/playwright/specs/states/checkbox-wrapper.vrt.ts

```diff
@@ -73,13 +68,10 @@ const scenario = async (page: Page, theme: Theme, scheme?: PrefersColorScheme):
   });
 
   await forceHoverState(page, '.hover p-checkbox-wrapper input[type="checkbox"]');
-  await forceHoverState(page, '.hover .force-label > p-checkbox-wrapper >>> span');
```

Comment:
Now this state isn't covered anywhere? Couldn't this be solved with better selectors or additional class names?